Cargo.toml

```diff
@@ -20,19 +20,26 @@ failure_derive = "0.1"
 gumdrop = "0.4"
 gumdrop_derive = "0.4"
 log = "0.4"
-rand = "0.4"
+rand = "0.5"
 serde = "1.0"
 serde_derive = "1.0"
+sha2 = "^0.7"
+hkdf = "^0.5"
+ring = "0.13"
 signatory = "0.6"
 simplelog = "0.5"
 toml = "0.4"
 yubihsm = { version = "0.14", optional = true }
-prost = { git = "https://github.com/Liamsi/prost", branch = "prost_amino_derive"}
-prost-derive = { git = "https://github.com/Liamsi/prost", branch = "prost_amino_derive"}
 serde_json = "1.0"
 hex = "0.3.2"
 bytes = "0.4"
 chrono = "0.4.2"
+byteorder = "^1.2"
+lazy_static = "1.1.0"
+x25519-dalek = "0.3"
```

Comment:
It looks like `x25519-dalek` enables its nightly feature (and vicariously `subtle`'s `nightly` feature) by default (which is probably a mistake/oversight):

https://github.com/dalek-cryptography/x25519-dalek/blob/2abfb37/Cargo.toml#L38

I think this will fix it:

```toml
x25519-dalek = { version = "0.3", default-features = false, features = ["std", "u64_backend"] }
```

Reply:
Thanks, that fixes that indeed (at least locally it works now).